contracts/root/stakeManager/IStakeManager.sol

```diff
@@ -1,6 +1,22 @@
-pragma solidity ^0.5.5;
+pragma solidity ^0.5.2;
 
 interface IStakeManager {
+  event Staked(address indexed user, uint256 indexed validatorId, uint256 indexed activatonEpoch, uint256 amount, uint256 total);
```

Comment:
No need to keep events in the interface. We can move these to the StakeManagerHeader or StakeManagerStorage